src/engraving/libmscore/score.cpp

```diff
@@ -5642,36 +5641,36 @@ void Score::createPaddingTable()
 
     // Ambitus
     for (auto& elem : _paddingTable[ElementType::AMBITUS]) {
-        elem.second = styleMM(Sid::ambitusMargin);
+        elem = styleMM(Sid::ambitusMargin);
     }
```

Comment:
If you decide to use my suggested definition of PaddingVector, then you could use the `fill` member of `std::array`: https://en.cppreference.com/w/cpp/container/array/fill
```c++
_paddingTable[ElementType::AMBITUS].fill(styleMM(Sid::ambitusMargin));
```
(in some other places below too).

Advantages:
- less typing/reading
- `styleMM(Sid::ambitusMargin)` is computed only once, instead of for every iteration